packages/overlays/src/OverlayController.js

```diff
@@ -891,97 +894,90 @@ export class OverlayController extends EventTargetShim {
 
   /**
    * Sets up backdrop on the given overlay. If there was a backdrop on another element
-   * it is removed. Otherwise this is the first time displaying a backdrop, so a fade-in
+   * it is removed. Otherwise this is the first time displaying a backdrop, so a animation-in
    * animation is played.
    * @param {{ animation?: boolean, phase: OverlayPhase }} config
    */
-  _handleBackdrop({ animation = true, phase }) {
-    if (this.placementMode === 'local') {
-      switch (phase) {
-        case 'init':
-          if (!this.backdropNode) {
-            this.__backdropNode = document.createElement('div');
-            /** @type {HTMLElement} */
-            (this.backdropNode).classList.add('local-overlays__backdrop');
-          }
-          this.backdropNode.slot = '_overlay-shadow-outlet';
-          /** @type {HTMLElement} */
-          (this.contentNode.parentNode).insertBefore(this.backdropNode, this.contentNode);
-          break;
-        case 'show':
-          this.__hasActiveBackdrop = true;
-          break;
-        case 'hide':
-          if (!this.backdropNode) {
-            return;
-          }
-          this.__hasActiveBackdrop = false;
-          break;
-        case 'teardown':
-          if (!this.backdropNode || !this.backdropNode.parentNode) {
-            return;
-          }
-          this.backdropNode.parentNode.removeChild(this.backdropNode);
-          this.__backdropNode = undefined;
-          break;
-        /* no default */
-      }
-      return;
-    }
+  _handleBackdrop({ phase }) {
     switch (phase) {
-      case 'init':
-        this.__backdropNode = document.createElement('div');
-        this.backdropNode.classList.add('global-overlays__backdrop');
-        /** @type {HTMLElement} */
-        (this.contentWrapperNode.parentElement).insertBefore(
-          this.backdropNode,
-          this.contentWrapperNode,
-        );
+      case 'init': {
+        if (!this.backdropNode) {
+          this.__backdropNode = document.createElement('div');
+          /** @type {HTMLElement} */
+          (this.backdropNode).classList.add(`${this.placementMode}-overlays__backdrop`);
+        }
+        this.backdropNode.slot = 'backdrop';
```

Comment:
Why should OverlayController know OverlayMixin.
It's cleaner if the connection is one way (OverlayMixins knows controller, not the other way around)

Reply:
I would actually like to remove this. It's leftover from the first commit, but I don't think this is needed anymore except when the user sets hasBackdrop: true and does not pass their own backdrop node (declaratively or imperatively), right?

Reply:
```js
if (!this.backdropNode) {
  this.__backdropNode = document.createElement('div');
  /** @type {HTMLElement} */
  (this.backdropNode).slot = 'backdrop';
  /** @type {HTMLElement} */
  (this.backdropNode).classList.add(`${this.placementMode}-overlays__backdrop`);
}
``` 
sound good?